eda_leda/implementacoes/sorting_algorithms/SelectionSort.py

```diff
@@ -0,0 +1,159 @@
+#coding: utf-8
+from Sort import Sort
+
+__author__ = "Ionesio Junior" 
+
+class SelectionSort(Sort):
+        """Implementação do algoritmo de Ordenação SelectionSort
+        
+        Classe para abstração do algoritmo de Ordenação SelectionSort implementado de forma iterativa e recursiva
+        Complexidade do Algoritmo: O(n^2)
+        """
+
+	def sort(self,array,leftIndex = 0 , rightIndex = None):
```

Comment:
Apenas por questões de estilo 😄 , se aceitar a sugestão, só clicar que ele já commita direto
```suggestion
	def sort(self, array, leftIndex = 0 , rightIndex = None):
```